sjb/config/test_cases/test_branch_image_registry_integration.yml

```diff
@@ -9,4 +9,4 @@ extensions:
       title: "run image-registry integration tests"
       repository: "image-registry"
       script: |-
-        OS_BUILD_ENV_TMP_VOLUME='/tmp' hack/env JUNIT_REPORT='true' make test-integration
```

Comment:
@stevekuznetsov has opinions on the use of `hack/env` so I will defer to him.

Reply:
`JUNIT_REPORT` is set to `true` by default

https://github.com/openshift/image-registry/blob/master/Makefile#L25